### PR TITLE
CE-3666 Add tracking of banner impressions

### PR DESCRIPTION
**Pull request by adamkarminski**
@Wikia/spitfires 

Tracking impression only if 100% of the banner is in viewport.

### CODE REVIEW

File: extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js [hunk 1]
```diff
@@ -41,14 +46,23 @@ require([
...
 			.on('click', '.pmp-entry-point-button', onEntryPointClick)
 			.on('click', '.pmp-entry-point-close', close);
+
+		bannerOffset = $banner.offset().top;
+		bannerHeight = $banner.outerHeight();
+
+		if (isEntryPointInViewport()) {
+			trackBannerImpression();
+		} else {
+			$(w).on('scroll', trackEntryPointInViewport);
```
Comment (lukaszkonieczny) [inline extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js]: What do you think about adding debounce function?

**Comment by lukaszkonieczny** [comment]
lgtm

File: extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js [hunk 2]
```diff
@@ -57,15 +71,35 @@ require([
...
+	function trackEntryPointInViewport() {
+		if (isEntryPointInViewport()) {
+			trackBannerImpression();
+			$(w).off('scroll.trackPMPEntryPoint');
+		}
+	}
+
+	function trackBannerImpression() {
+		track({
+			label: '',
```
Comment (Grunny) [inline extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js:91]: Why is the label empty?

Comment (adamkarminski) [inline extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js:91]: I'm working on tracking viewability of the banner. It's just a placeholder and if I don't use it in the end I'll remove it in the upcoming PR. What do you think? :)

Comment (Grunny) [inline extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js:91]: Sure.

**Pull request merged**